gossip/simulation/network.go

```diff
@@ -76,13 +76,12 @@ func NewNetwork(nodeCount int, networkType string,
 	} else {
 		bootstrap = addrs[:3]
 	}
-
 	nodes := make([]*Node, nodeCount)
 	for i := 0; i < nodeCount; i++ {
-		node := gossip.New(rpcContext, gossipInterval, gossipBootstrap)
+		// Pass an empty string to gossipBootstrap and set it later.
+		node := gossip.New(rpcContext, gossipInterval, "")
```

Comment:
Sure, one option might be

- change `gossip.New` to take a slice of ```net.Addr``` as bootstrap addresses instead of a string.
- change `network.go` to pass `bootstrap` to `gossip.New` instead of calling `SetBootstrap`.